app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Settings.kt

```diff
@@ -0,0 +1,56 @@
+package org.fossasia.susi.ai.rest.responses.susi
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Created by meeera on 30/6/17.
+ */
+
+class Settings {
+
+    /**
+     * Gets speechAlways
+     *
+     * @return the speechAlways
+     */
```

Comment:
Avoid comments here, these are specific to java not kotlin. We are not using any getters and setters here.